tests/chaos/oadp/conftest.py

```
@@ -79,3 +80,80 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+            content=TEXT_TO_TEST,
+            stop_vm=False,
+        )
+
+        return vm
+
+    yield _create
+
+    # Cleanup all created VMs
+    for vm, vm_generator in created_vms:
+        try:
+            vm_generator.__exit__(None, None, None)  # noqa: FCN001
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `FCN001`)

Remove unused `noqa` directive

(RUF100)